### PR TITLE
More CI/testing updates

### PR DESCRIPTION
**Pull request by kkiesling**
As I am learning more and more about CI, testing, and pytest, I realized there are updates needed here. This PR does the following:
1. remove the pip install line from CI (unnecessary)
2. Changes the paths of the .h5m test files to be relative to the top level directory so that pytest can be run from the top level
3. runs pytest from the top level directory on CI

### CODE REVIEW

**Comment by kkiesling** [comment]
> I think it is fine to remove `pip install -e . --user`.
> But I would suggest to install it using pip (with out the `-e`) and try to import it from the `ROOT` folder.

I don't quite understand what you are suggesting

**Comment by bam241** [comment]
basically having a job checking the proper installation of the package, and that you can import it

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
 
 jobs:
+    install_py3:
+        docker:
+            - image: svalinn/pymoab-py3-18.04
+        steps:
+            - checkout
+            - run:
+                command: pip install . --user
+                command: python -c "import dagmc_stats"
```
Comment (bam241) [inline .circleci/config.yml:11]: could you do that in `$root` ? not the repo folder ?

**Comment by bam241** [comment]
LGTM! thx @kkiesling 

Merging on test completion

**Pull request merged**